tests-example/tilt_modules/tests

```diff
@@ -0,0 +1 @@
+/Users/maia/code/go/src/github.com/tilt-dev/tilt-extensions/tests
```

Comment:
hmmmm...is this a symlink? i don't think this will work properly if you check it in